problem_builder/mentoring.py

```diff
@@ -236,7 +236,9 @@ def student_view(self, context):
 
         for child_id in self.children:
             child = self.runtime.get_block(child_id)
-            if not isinstance(child, MentoringMessageBlock):
+            if child is None:  # child should not be None but it can happen due to bugs or permission issues
+                child_content += u"<p>[{}]</p>".format(self._(u"Error: Unable to load child component."))
+            elif not isinstance(child, MentoringMessageBlock):
```

Comment:
Good point, but since there's no mechanism for setting permissions on child blocks at the moment, I think it would indicate an error, at least with the current runtime.

BTW, does the API explicitly state whether the "children" property may include blocks that the user doesn't have permission to access?